### PR TITLE
dependency and build updates to remove npm vulnerability #24

### PR DESCRIPTION
**Pull request by smm-telus**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,31 +1,50 @@
...
   entry: './src/index',
-  module: {
-    loaders: [
-      { test: /\.json$/, loader: 'json' },
-      { test: /\.js$/, loader: 'babel', exclude: /node_modules/ }
-    ]
-  },
   output: {
-    filename: 'dist/markdown-react.min.js',
+    filename: 'markdown-react.min.js',
```
Comment (kof) [inline webpack.config.js:7]: why removing dist?

Comment (smm-telus) [reply webpack.config.js:7]: I believe Webpack 4 automagically generates the `dist ` folder for you unless you say otherwise.

Comment (kof) [reply webpack.config.js:7]: did you try? If yes I am fine with merging.

Comment (smm-telus) [reply webpack.config.js:7]: I deleted the `dist` folder and re-ran `npm i` as well as `npm run dist`. It generated the `dist` folder automatically.

**Pull request merged**